### PR TITLE
dev and product release notes 20231229

### PR DESCRIPTION
**Pull request by Nick011**
Dev and product release notes for December 29th, 2023

### CODE REVIEW

File: updates/development-updates/december-29th-2023.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+* Improved User Settings page with optional email invites
+ 
+</details>
+
+<details>
+
+<summary><strong>Bug fixes and chores</strong></summary>
+
+* Auto genearte org slugs on org creation
```
Comment (jtwillis92) [inline updates/development-updates/december-29th-2023.md:19]: Typo here

File: updates/development-updates/december-29th-2023.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+</details>
+
+<details>
+
+<summary><strong>In review, testing and development</strong></summary>
+
+* SentinelOne integration refactor
+* Improved Time Saved functionality with Task level Time Saved and persistence beyond workflow execution retention
+* Bug fixes for App Platform
```
Comment (jtwillis92) [inline updates/development-updates/december-29th-2023.md:30]: Might be worth mentioning the Microsoft improvements vaguely

**Review by jtwillis92 — APPROVED** [review]
Just a couple nits

File: updates/development-updates/december-29th-2023.md (hunk 1 above)
Comment (Nick011) [reply updates/development-updates/december-29th-2023.md]: Fixed

File: updates/development-updates/december-29th-2023.md (hunk 2 above)
Comment (Nick011) [reply updates/development-updates/december-29th-2023.md:31]: Good suggestion, added.

**Pull request merged**